packages/core/src/external/carequality/ihe-gateway-v2/ihe-gateway-v2-async.ts

```diff
@@ -17,7 +17,7 @@ dayjs.extend(duration);
 const SLEEP_IN_BETWEEN_DOCUMENT_RETRIEVAL_REQUESTS = dayjs.duration({ seconds: 1 });
 const MAX_GATEWAYS_BEFORE_CHUNK = 1000;
 const MAX_DOCUMENT_QUERY_REQUESTS_PER_INVOCATION = 20;
-const MAX_DOCUMENT_RETRIEVAL_REQUESTS_PER_INVOCATION = 20;
+const MAX_DOCUMENT_RETRIEVAL_REQUESTS_PER_INVOCATION = 10;
```

Comment:
Might make sense to ultimately flatten this logic so that there's a single request count per lambda invocation.

Reply:
I think this would be especially good for DR, yeah.

I see the benefit of grouping a bunch of XCPDs in a single lambda request (no much memory/timeout issue), but to download a file, a single lambda invocation seem to make sense.

Reply:
ok I think this concern needs more consideration and is beyond the scope of the PR. There are a bunch of other considerations to take into account: 
1. cost
2. concurrency limits
3. complete reworking of querrying architecture to be queue based

Reply:
Moving DRs to individual ones has nothing to do with (3) and (2).

For (2), we have 1,000 concurrent lambda executions and we can ask to increase this limit to 10's of 1,000 ([source](https://docs.aws.amazon.com/lambda/latest/dg/gettingstarted-limits.html)).

For. (1)/cost, this might impact, but it would make each single invocation shorter - and lambdas are cheap, it doesn't even make it to the top 10 costs we have.